### PR TITLE
Cleans up a buncha messed up spans on the plant analyzer

### PR DESCRIPTION
**Pull request by MrMelbert**
## About The Pull Request

Cleans up some of the spans on the plant analyzer from the mass span conversion PR. Find and replace did not do wonders for the plant analyzer formatting (which is probably partially my fault).

Also adjusts some visible_message formatting and some text formatting of the plant analyzer.

Fixed formatting:
image

## Why It's Good For The Game

Makes the plant analyzer readout consistent in its formatting again.

## Changelog
:cl: Melbert
fix:  Plant Analyzer's readout should look less weird and more consistent again.
/:cl:

### CODE REVIEW

File: code/modules/hydroponics/hydroitemdefines.dm [hunk 1]
```diff
@@ -160,15 +163,13 @@
  * Returns the formatted message as text.
  */
 /obj/item/plant_analyzer/proc/scan_tray_chems(obj/machinery/hydroponics/scanned_tray)
-	var/returned_message = "<span class='info'>*---------*\n"
+	var/returned_message = "*---------*\n"
 	if(scanned_tray.myseed)
 		returned_message += "*** <B>[scanned_tray.myseed.plantname]</B> ***\n"
```
Comment (coiax) [inline code/modules/hydroponics/hydroitemdefines.dm:168]: Should this be `span_bold` as well?

File: code/modules/hydroponics/hydroitemdefines.dm [hunk 2]
```diff
@@ -211,7 +212,7 @@
  * Returns the formatted output as text.
  */
 /obj/item/plant_analyzer/proc/scan_plant_chems(obj/item/scanned_object)
-	var/returned_message = "[span_info("*---------*\nThis is \a <span class='name'>[scanned_object]")].\n"
+	var/returned_message = "*---------*\nThis is a [span_name("[scanned_object.name]")].\n"
```
Comment (coiax) [inline code/modules/hydroponics/hydroitemdefines.dm:215]: Does this work gramatically when you are scanning apples?

> This is a apple.

**Review by coiax — CHANGES_REQUESTED** [review]

File: code/modules/hydroponics/hydroitemdefines.dm (hunk 1 above)
Comment (MrMelbert) [reply code/modules/hydroponics/hydroitemdefines.dm]: Might as well.

File: code/modules/hydroponics/hydroitemdefines.dm (hunk 2 above)
Comment (MrMelbert) [reply code/modules/hydroponics/hydroitemdefines.dm]: You're correct about that, for some reason I noticed it in testing but thought `a apple` was fine.

**Pull request merged**